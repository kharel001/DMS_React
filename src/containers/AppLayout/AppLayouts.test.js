import React from 'react';
import AppLayouts from './index';
import {
    configure,
    shallow
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})
const setup = (props = {}, state = null) => {
    return (shallow( < AppLayouts
     {...props}  />)
    )
}


describe('<AppLayouts />', () => {
    

it("AppLayouts render", () => { 
        const wrapper= setup();
        expect(wrapper).toMatchSnapshot();
})
})
