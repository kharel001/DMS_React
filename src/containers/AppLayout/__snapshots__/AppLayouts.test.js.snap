// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppLayouts /> AppLayouts render 1`] = `ShallowWrapper {}`;
